Jenkinsfile

```diff
@@ -10,11 +10,9 @@
  *  SPDX-License-Identifier: MPL-2.0# Copyright (C) 2017 Pelagicore AB
  */
 
-
-node ("PyTest") {
+node ("PyTest && Vagrant && VirtualBox") {
     stage("Checkout") {
         checkout scm
-        sh 'git submodule update --init'
```

Comment:
Is there a reason to remove this line? :)

Reply:
There are no submodules required by this repo

Reply:
Yup that is true, but it doesn't hurt does it, can you comment @thenor ? I mean it creates a bigger diff against other jenkinsfiles and we might add a submodule at some point :). 

Furthermore, even if we include this change it should be a separate commit due to the fact that changes need to be atomic.